src/code_memory/server.py

```
@@ -0,0 +1,199 @@
+        logger.info("Shutting down code memory server...")
+        await storage.cleanup()
+        await embedder.cleanup()
+
+
+# Create server with lifecycle management
+mcp = FastMCP("Code Memory Server", lifespan=app_lifespan)
+
+
+@mcp.tool()
+async def store_code_snippet(
+    code: str, 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)